src/applications/mhv/secure-messaging/tests/e2e/pages/PatientComposePage.js

```diff
@@ -64,7 +68,7 @@ class PatientComposePage {
       .click({ force: true });
     // this.attachMessageFromFile('test_image.jpg');
     this.getMessageSubjectField().type('Test Subject');
-    this.getMessageBodyField().type('Test message body');
```

Comment:
@fazilqa @haniasma this is a temp fix. If you guys can look deeper into what is happening. I can tell that it is caused by the function that we run when entering the field to set the cursor to initial position. But I am not sure what is a proper way for cypress to handle. Temporarily placing `{force:true}` here